AGENTS.md

```
@@ -1,2 +1,2 @@
-- After making any changes always execute `pnpm build && pnpm test:coverage` to verify
+- After making any changes always execute `pnpm format && pnpm build && pnpm test` to verify
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)